tests/integration/cairo1_hint_locals/call/call_test.cairo

```diff
@@ -112,41 +77,15 @@ fn test_call_cairo0_non_existing_entrypoint() {
 
 #[test]
 fn test_call_wrong_name() {
-    let class_hash = declare('simple').unwrap();
-    assert(class_hash != 0, 'class_hash != 0');
-
-    let prepare_result = prepare(class_hash, ArrayTrait::new()).unwrap();
-
-    assert(prepare_result.contract_address != 0, 'deployed contract_address != 0');
-    assert(prepare_result.class_hash != 0, 'deployed class_hash != 0');
-
-    let prepared_contract = PreparedContract {
-        contract_address: prepare_result.contract_address,
-        class_hash: prepare_result.class_hash,
-        constructor_calldata: prepare_result.constructor_calldata
-    };
-    let deployed_contract_address = deploy(prepared_contract).unwrap();
+    let deployed_contract_address = deploy_contract('simple', ArrayTrait::new()).unwrap();
     assert(deployed_contract_address != 0, 'deployed_contract_address != 0');
 
     call(deployed_contract_address, 'empty_no_exist', ArrayTrait::new()).unwrap();
 }
 
 #[test]
 fn test_call_wrong_number_of_args() {
```

Comment:
Should call just pass when there args are invalid in protostar? Is this behavior documented somewhere? 

I've tried making call with too many arguments on testnet and got 

```json
{"code": "StarknetErrorCode.TRANSACTION_FAILED", "message": "Error at pc=0:104:\nGot an exception while executing a hint.\nCairo traceback (most recent call last):\nUnknown location (pc=0:1678)\nUnknown location (pc=0:1664)\n\nError in the called contract (0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7):\nError at pc=0:769:\nAn ASSERT_EQ instruction failed: 11:1 != 11:2."}
```

Reply:
Discussed offline, misunderstanding.